Model/EventsForThirdPartyModules.php

```diff
@@ -62,6 +63,13 @@ class EventsForThirdPartyModules
                     "sendClasses" => ["Aheadworks\Giftcard\Api\GiftcardCartManagementInterface"],
                     "boltClass" => Aheadworks_Giftcard::class,
                 ],
+                [
+                    "module" => "Mirasvit_Credit",
+                    "sendClasses" => ["Mirasvit\Credit\Helper\Data",
+                                      "Mirasvit\Credit\Service\Calculation",
+                                      "Mirasvit\Credit\Api\Config\CalculationConfigInterface|Mirasvit\Credit\Service\Config\CalculationConfig",],
```

Comment:
Could you add a comment about last and legacy version

Reply:
Comment is added there https://github.com/BoltApp/bolt-magento2/pull/920/files#diff-54ef5f36c96f487f90a039490c80b7d5R152